content-src/components/Topics/Topics.jsx

```diff
@@ -19,3 +19,30 @@ export class Topics extends React.PureComponent {
     );
   }
 }
+
+export class TopicsHeader extends React.PureComponent {
```

Comment:
The previous topics component was build to work inside a cards section, so I felt forking here was logical.